docs/usermanual.md

```diff
@@ -544,3 +545,94 @@ To ensure consistent Markdown formatting run the following:
 # Use git-bash on Windows
 npm run prettier:text
 ```
+
+### Swagger
+
+In order to run Swagger, you need to execute the IoT Agent
+(https://github.com/telefonicaid/iotagent-ul/blob/master/docs/installationguide.md#usage) and then you can access to:
```

Comment:
Missing square brackets for link: 
```md
[IoT Agent](http://etc.)
```

Reply:
fixed in 53b15a6eccc20866ec89dd363d57d95513e620f0